Misc/NEWS

```diff
@@ -10,6 +10,8 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-31061: Fixed a crash when using asyncio and threads.
```

Comment:
Last nit: the change in Misc/NEWS needs to be reverted, we need only blurb file.